src/tree/tree.scss

```diff
@@ -0,0 +1,148 @@
+// Copyright (c) Bentley Systems, Incorporated. All rights reserved.
+// See LICENSE.md in the project root for license terms and full copyright notice.
+@import '../style/index';
+@import '../icon/index';
+
+$iui-active-outline: thin solid t(iui-color-foreground-primary);
+$iui-child-node-spacing: calc(36px * (var(--level, 0)));
+
+@mixin iui-tree {
+  @include themed {
+    background-color: t(iui-color-background-1);
+  }
+}
+
+@mixin iui-tree-node {
+  @include iui-focus;
+  display: flex;
+  width: 100%;
+  align-items: center;
+  padding: $iui-xxs;
+  cursor: pointer;
+  box-sizing: border-box;
+  user-select: none;
+  margin-right: $iui-sm;
+  //min-height: $iui-component-height-small + (2 * $iui-xxs); //button height + padding
+  min-height: $iui-baseline * 3;
+
+  > .iui-icon {
+    @include iui-icons-default;
+    margin-right: $iui-s;
+    margin-left: $iui-sm + 1;
```

Comment:
Why you need margin left here? Especially why +1?

Reply:
Removed

Reply:
It was more a question then asking to remove 😀 I think layout is broken now because of that and tests are failing:
![image](https://user-images.githubusercontent.com/36186912/143186777-47f7e990-151a-4057-9289-92ba08f429cd.png)

Reply:
I added iui-node-content class to simplify margins and this should be fixed now

Reply:
Yeah but I still don't understand why you need right and left margins on different elements. I think appropriate `margin-right` on each element should be enough.

Reply:
Additional margins are needed on some elements to help align with other elements above and below. 

For example `.iui-icon` has left margin so that it aligns with the icon inside the expander button, without this it aligns with the button but not the icon inside the button
![image](https://user-images.githubusercontent.com/88331924/143948415-5f6aff9d-c88d-4105-8c8b-f1c576c246b2.png)

Reply:
Updated this margin